### PR TITLE
Adding Python 3 support

### PR DESCRIPTION
**Pull request by lexhung**
Tests run successfully with Python 2.7 & Python 3.5.

### CODE REVIEW

File: 2to3.diff [hunk 1]
```diff
@@ -0,0 +1,432 @@
+--- ./decorators.py (original)
```
Comment (lucaswiman) [inline 2to3.diff]: Did you mean to include this?

**Comment by lucaswiman** [comment]
Maybe also update the [trove classifiers](https://github.com/azoner/pyx12/blob/4e004812113635bc12e54b473bad5c98333ac541/setup.py#L39) so the package shows as Python 3 compatible. 

```python
    classifiers=[
        'Development Status :: 5 - Production/Stable',
        'Environment :: Console',
        'Intended Audience :: Healthcare Industry',
        'Intended Audience :: Developers',
        'License :: OSI Approved :: BSD License',
        'Programming Language :: Python',
        'Programming Language :: Python',
        'Programming Language :: Python :: 2',
        'Programming Language :: Python :: 2.7',
        'Programming Language :: Python :: 3',
        'Programming Language :: Python :: 3.4',
        'Topic :: Office/Business',
        'Topic :: Software Development :: Libraries :: Python Modules',
    ]
```

**Comment by lucaswiman** [comment]
@azoner Any chance of getting this merged and deployed to Pypi? This would be very useful for people either starting new projects or in upgrading older codebases to Python 3.

**Comment by azoner** [comment]
Totally agree - just need to find some time

File: 2to3.diff (hunk 1 above)
Comment (azoner) [inline 2to3.diff]: nope - will delete

Comment (azoner) [inline 2to3.diff]: was using dump_args decorator.  restored file

**Pull request merged**